api/server.js

```diff
@@ -0,0 +1,18 @@
+const cors = require('cors');
+const express = require('express');
+const app = express()
+const port = 8000
+
+app.use(cors());
+
+app.get("/", (req, res) => {
+    res.send("Welcome to the homepage!")
+})
+
+app.post("/", (req, res) => {
+    res.send("Post worked!")
+})
+
+app.listen(port, () => {
+    console.log(`Listening at port ${port}`)
+})
```

Comment:
You are also confusing server and app. Your entry point is an app and should be listening for request and have the API route connectors. Better use one file if you are unsure how to separate this.